config/component_macros.cmake

```diff
@@ -160,6 +160,13 @@ or the target must be labeled NOEXPORT.")
     set( ace_FOLDER ${comp_target} )
   endif()
 
+  # Helper for clang-tidy that points to very old gcc STL files.  We need STL
```

Comment:
Move this into a cmake-function an call it instead of copy-and-paste into multiple locations.